opm/material/densead/MathV.hpp

```diff
@@ -0,0 +1,512 @@
+// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```

Comment:
if you implement my suggestion from above, this file should not be necessary. maybe the normal Math.hpp needs a few modifications, though.